recipes/tinyxml2/all/conanfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class Tinyxml2Conan(ConanFile):
+    name = "tinyxml2"
+    description = "Simple, small, efficient, C++ XML parser that can be " \
+                  "easily integrated into other programs."
+    license = "Zlib"
+    topics = ("conan", "tinyxml2", "xml", "parser")
+    homepage = "https://github.com/leethomason/tinyxml2"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
conan automatically adds this flag if a shared option is detected.

Reply:
fixed